Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -76,8 +78,16 @@ public static bool AreFoldersAvailable
 
         static MixedRealityToolkitFiles()
         {
-            string path = Application.dataPath;
-            searchForFoldersTask = Task.Run(() => SearchForFoldersAsync(path));
+            if (AssetDatabase.IsValidFolder($"Packages/{MRTK_PACKAGE_ID}"))
+            {
+                mrtkFolders.Add(Path.GetFullPath($"Packages/{MRTK_PACKAGE_ID}"));
+                searchForFoldersTask = Task.CompletedTask;
+            }
+            else
+            {
+                string path = Application.dataPath;
+                searchForFoldersTask = Task.Run(() => SearchForFoldersAsync(path));
+            }
```

Comment:
The change I made below was to account for potentially split folders, but I guess that may have been excessive. This seems reasonable.

Reply:
Is this resolved? Or do I need to change this to have everything in one UPM package and then go into the individual folders? I tried to match the Nuget Package model that seems to be in the repo right now

Reply:
Yes, it is, sorry!

Reply:
Looks like a bunch more stuff was added to mixedrealitytoolkitfiles to support Core and other folders. I merged my change with this and am testing it right now